src/khan.ts

```diff
@@ -0,0 +1,133 @@
+import net from 'net'
+import { Cell, jam, dwim, Noun, enjs } from '@urbit/nockjs'
+import { newtDecode, newtEncode } from './bytes'
+
+// returns a js Noun Object with the structure of a %fyrd
+function Fyrd(desk: string, thread: string, mark_out: string, mark_in: string, atom: string) {
+  return dwim([
```

Comment:
this is more concise, but it hides the structure of what Fyrd() is constructing. is dwim documented somewhere?